src/__tests__/Button.spec.js

```diff
@@ -5,11 +5,15 @@ import Adapter from 'enzyme-adapter-react-16';
 
 import Button from '../components/Button/Button';
 
-Enzyme.configure({ adapter: new Adapter() });
-
 describe('<Button />', () => {
   it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<Button />, div);
+    shallow(<Button />);
+  });
+
+  it('should render a button a value of its "name" prop', () => {
+    const root = shallow(<Button name="name" />);
+    const instance = root.instance();
+
+    expect(root.find('button').text()).toEqual('name');
```

Comment:
This is a good test to determine whether React is capable of creating a DOM element with the attribute you prescribed it or not. If this failed, you could then immediately go to the React github and create an issue.